src/chains/tests/test_views.py

```diff
@@ -80,6 +80,8 @@ def test_json_payload_format(self) -> None:
                     ],
                     "ensRegistryAddress": chain.ens_registry_address,
                     "recommendedMasterCopyVersion": chain.recommended_master_copy_version,
+                    "pricesProviderNativeCoin": chain.prices_provider_native_coin,
+                    "pricesProviderChainName": chain.prices_provider_chain_name,
```

Comment:
What do you think about nesting these, e.g.
```
{
  pricesProvider: {
    nativeCoin: ...,
    chainName: ...,
}
```

Reply:
Agreed! Done in [ed93b59](https://github.com/safe-global/safe-config-service/pull/1124/commits/ed93b59716d076d824ceb4f8559358172052bb55)